ISL.Providers.PDS.FakeFHIR/Mappers/PatientMapper.cs

```diff
@@ -96,7 +96,7 @@ internal static Patient FromPdsPatientDetails(PdsPatientDetails pdsPatientDetail
                 {
                     new ContactPoint(
                         system: ContactPoint.ContactPointSystem.Phone,
-                        use: ContactPoint.ContactPointUse.Home,
+                        use: ContactPoint.ContactPointUse.Mobile,
```

Comment:
Feels like this PR needs unit tests.   Nothing will catch that you have swopped fields.